### PR TITLE
Fix Notebook e2e Tests

### PR DESCRIPTION
**Pull request by VaishnaviHire**
This PR fixes the issue with accessing e2e script. Moved all the required commands under the `e2e` test job

### CODE REVIEW

File: ci-operator/config/opendatahub-io/kubeflow/opendatahub-io-kubeflow-master.yaml [hunk 1]
```diff
@@ -47,7 +47,24 @@ tests:
...
+        unset GOFLAGS
+        pushd components/odh-notebook-controller
+        oc new-project odh-notebook-controller-system
+        IFS=':' read -r -a CTRL_IMG <<< "${ODH_NOTEBOOK_CONTROLLER_IMAGE}"
+        export IMG="${CTRL_IMG[0]}"
+        export TAG="${CTRL_IMG[1]}"
+        export K8S_NAMESPACE="odh-notebook-controller-system"
+        cp $KUBECONFIG /tmp/kubeconfig
+        chmod 644 /tmp/kubeconfig
+        export KUBECONFIG=/tmp/kubeconfig
```
Comment (kpouget) [inline ci-operator/config/opendatahub-io/kubeflow/opendatahub-io-kubeflow-master.yaml:60]: you should try to make your test code resilient to the read-only `KUBECONFIG`, the scripts will be safer to use. This means passing `--skip-config-write` to `oc new-project` and always using `-n $namespace`.

Comment (LaVLaS) [reply ci-operator/config/opendatahub-io/kubeflow/opendatahub-io-kubeflow-master.yaml:60]: Agreed.  We have added that as a item on plans to update the e2e test suite

**Review by LaVLaS — APPROVED** [review]
/lgtm

**Pull request merged**